src/shell/scripts/omp.ps1

```diff
@@ -111,12 +111,19 @@ New-Module -Name "oh-my-posh-core" -ScriptBlock {
     }
 
     function Enable-PoshTransientPrompt {
-        Set-PSReadlineKeyHandler -Key Enter -ScriptBlock {
-            $script:TransientPrompt = $true
+        Set-PSReadLineKeyHandler -Key Enter -ScriptBlock {
             $previousOutputEncoding = [Console]::OutputEncoding
-            [Console]::OutputEncoding = [Text.Encoding]::UTF8
             try {
-                [Microsoft.PowerShell.PSConsoleReadLine]::InvokePrompt()
+                $ast = $null
+                $tokens = $null
+                $parseErrors = $null
```

Comment:
shouldn't we only need this variable and add the other `$null` inline?

Reply:
🆗 Changed.